### PR TITLE
Memsat svm 1.3

### PR DESCRIPTION
**Pull request by sjanssen2**
A recipe for the program memsat-svm version 1.3

### CODE REVIEW

File: recipes/memsat-svm/reloc.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+--- run_memsat-svm.pl	2013-09-09 09:18:43.000000000 -0700
++++ run_memsat-svm.reloc.pl	2017-02-24 11:15:27.253626507 -0800
+@@ -22,16 +22,16 @@
+ 
+ ## IMPORTANT : this variable must be set here to the absolute path to the
+ ## library called "lib" and found in the same directory as this script.
+-use lib 'lib';
++use lib '/opt/anaconda1anaconda2anaconda3/share/memsat-svm/lib/';
```
Comment (qiyunzhu) [inline recipes/memsat-svm/reloc.patch:8]: The directory name "anaconda1anaconda2anaconda3" looks like a type to me?

File: recipes/memsat-svm/badfd.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+--- run_memsat-svm.pl	2013-09-09 09:18:43.000000000 -0700
++++ run_memsat-svm.badfd.pl	2017-02-21 09:24:58.516103503 -0800
+@@ -388,8 +388,8 @@
+ 		my $tmp_fasta = "$tmp_rootpath.fasta";
+ 		my $tmp_chk = "$tmp_rootpath.chk";
+ 		my $system = `cp -f $fasta $tmp_fasta`;
+-		print "$ncbidir/blastpgp -a $cores -j 2 -h 1e-3 -e 1e-3 -b 0 -d $dbname -i $tmp_fasta -C $tmp_chk >& $blast_out\n\n";
```
Comment (qiyunzhu) [inline recipes/memsat-svm/badfd.patch:7]: Is there actually any difference before and after?

Comment (qiyunzhu) [reply recipes/memsat-svm/badfd.patch:7]: Also the BLAST parameters look very relaxed to me.

**Review by qiyunzhu — COMMENTED** [review]
All look good! No problems. Just three questions.

File: recipes/memsat-svm/reloc.patch (hunk 1 above)
Comment (sjanssen2) [reply recipes/memsat-svm/reloc.patch:8]: actually /opt/anaconda1anaconda2anaconda3/ is not a typo but the actual keyword for conda relocation mechanism

File: recipes/memsat-svm/badfd.patch (hunk 2 above)
Comment (sjanssen2) [reply recipes/memsat-svm/badfd.patch:7]: >& and &> make a different in some Linux flavours, e.g. on Travis
BLAST parameters are set by the authors of this program and I cannot comment on their rational.

**Comment by qiyunzhu** [comment]
@sjanssen2 Looks good!

**Comment by sjanssen2** [comment]
Thanks Qiyun. You are amazing. Should you be bored over the weekend - 
here is one more https://github.com/biocore/conda-recipes/pull/92 And 
since you are becoming an expert by all your reviewing activity, your 
feedback his highly appreciated!

Thanks,
Stefan

**Comment by qiyunzhu** [comment]
Hello @sjanssen2 I reviewed your PR #92. You are always welcome!

**Comment by sjanssen2** [comment]
Good morning. @qiyunzhu @RNAer @mortonjt could you give this PR a few minutes of your time and eventually merge?

**Comment by qiyunzhu** [comment]
Hi @sjanssen2 I think this PR is good to ship. @mortonjt @RNAer what do you think?

**Comment by sjanssen2** [comment]
Thanks @mortonjt !

**Pull request merged**